packages/ckeditor5-list/src/todolist/todolistediting.ts

```diff
@@ -111,6 +111,12 @@ export default class TodoListEditing extends Plugin {
 			dispatcher.on( 'element:span', elementUpcastConsumingConverter(
 				{ name: 'span', classes: 'todo-list__label__description' }
 			) );
+
+			// Priority is set to low to allow generic list item converter to run first.
+			dispatcher.on( 'element:li', todoListItemUpcastConverter(), {
+				priority: 'low'
+			} );
+
```

Comment:
Please move this converter above the "consuming" converters so it is not lost in the crowd of just consuming converters.